oidc-sso-sample/Dispatch/pom.xml

```diff
@@ -2,7 +2,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <parent>
-        <groupId>org.wso2.is.oidc</groupId>
+        <groupId>org.wso2.is.samples</groupId>
```

Comment:
We may not be able to use org.wso2.is.* in the group id pattern, as it may conflict with product-is artifacts. Can you check?